src/languages/cpp.js

```diff
@@ -402,18 +396,22 @@ export default function(hljs) {
   ];
 
   const CPP_KEYWORDS = {
-    type: [...RESERVED_TYPES, ...TYPES],
+    type: RESERVED_TYPES,
     keyword: RESERVED_KEYWORDS,
     built_in: BUILT_INS,
-    literal: LITERALS
+    literal: LITERALS,
+    keywords: {
+      // Only for relevance, not highlighting.
+      _hint: TYPE_HINTS
+    }
```

Comment:
```suggestion
    literal: LITERALS,
    _type_hints: TYPE_HINTS
```

You're already inside a keywords object here, are you not?

Reply:
Right, fixed it. Got an error with the Arduino highlighting markup test now though -- it seems all highlighting is missing from the result.

Reply:
Fixed this error as well, just a sec..

Reply:
(test is fixed now as well)